pyt/cfg/stmt_visitor.py

```diff
@@ -1046,55 +1051,69 @@ def visit_Import(self, node):
+            # e.g. we want a call to "os.system" be recognised, even if we do "import os as myos"
+            if alias.asname is not None and alias.asname != alias.name:
+                local_definitions = self.module_definitions_stack[-1]
+                local_definitions.import_alias_mapping[name.asname] = alias.name
             if alias.name not in uninspectable_modules:
-                log.warn("Cannot inspect module %s", alias.name)
+                log.warning("Cannot inspect module %s", alias.name)
                 uninspectable_modules.add(alias.name)  # Don't repeatedly warn about this
         return IgnoredNode()
 
     def visit_ImportFrom(self, node):
         # Is it relative?
         if node.level > 0:
             return self.handle_relative_import(node)
-        else:
-            for module in self.local_modules:
-                if node.module == module[0]:
-                    if os.path.isdir(module[1]):
-                        return self.from_directory_import(
-                            module,
-                            not_as_alias_handler(node.names),
-                            as_alias_handler(node.names)
-                        )
-                    return self.add_module(
+        # not relative
+        for module in self.local_modules:
+            if node.module == module[0]:
+                if os.path.isdir(module[1]):
+                    return self.from_directory_import(
                         module,
-                        None,
-                        as_alias_handler(node.names),
-                        retrieve_import_alias_mapping(node.names),
-                        from_from=True
+                        not_as_alias_handler(node.names),
+                        as_alias_handler(node.names)
                     )
-            for module in self.project_modules:
-                name = module[0]
-                if node.module == name:
-                    if os.path.isdir(module[1]):
-                        return self.from_directory_import(
-                            module,
-                            not_as_alias_handler(node.names),
-                            as_alias_handler(node.names),
-                            retrieve_import_alias_mapping(node.names)
-                        )
-                    return self.add_module(
+                return self.add_module(
+                    module=module,
+                    module_or_package_name=None,
+                    local_names=as_alias_handler(node.names),
+                    import_alias_mapping=retrieve_import_alias_mapping(node.names),
+                    from_from=True
+                )
+        for module in self.project_modules:
+            name = module[0]
+            if node.module == name:
+                if os.path.isdir(module[1]):
+                    return self.from_directory_import(
                         module,
-                        None,
```

Comment:
The main thing I'm unsure about is, why I was passing `None` for `real_names`, but it was probably wrong :)

I do remember I wrote a [lot](https://github.com/python-security/pyt/pull/23/files) of [tests](https://github.com/python-security/pyt/pull/39/files) in the [import PRs](https://github.com/python-security/pyt/pull/29/files), so I'm somewhat confident this code is well covered. (Granted it was when I was an even worse engineer than I am now 😁 ).